webpages/popup-loader.js

```diff
@@ -121,7 +122,7 @@ async function refetchSession(addon) {
   });
 
   scratchAddons.methods.getEnabledAddons = (tag) =>
-    sendMessage({
+    chrome.runtime.sendMessage({
```

Comment:
You missed this use of `sendMessage()` - that's the reason for the error discovered by @Samq64.